holmes/plugins/toolsets/newrelic/newrelic.py

```
@@ -0,0 +1,242 @@
+            for rec in copied:
+                # Keep only fields that aren’t common (don’t mutate the original record)
+                entry = {k: v for k, v in rec.items() if k not in common_fields}
+                data_entries.append(entry)
+
+            group: Dict[str, Any] = dict(common_fields)
+            if "data" in group:
+                group["_common.data"] = group.pop("data")
+
+            group["data"] = data_entries
+
+            return [group]
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)